### ISSUE TITLE
bad custom fields should default to beginner values

### ISSUE THREAD

**Issue by Namdrib** [Namdrib/java-minesweeper#20]
in Minesweeper.rbMenuItem.addActionListener, entering a bad input for heightField, widthField or minesField (e.g. empty strings, non-numerical input, etc.) should result in beginner values for each incorrect field.

![2018-12-01_18-31-29](https://user-images.githubusercontent.com/9072806/49325737-57ff0e00-f597-11e8-9832-b076de2a774d.gif)

```java
          try {
            newHeight = Integer.parseInt(heightField.getText());
          } catch (NumberFormatException ex) {
            newHeight = 9;
          }
          try {
            newWidth = Integer.parseInt(widthField.getText());
          } catch (NumberFormatException ex) {
            newWidth = 9;
          }
          try {
            newMines = Integer.parseInt(minesField.getText());
          } catch (NumberFormatException ex) {
            newMines = 10;
          }
```

This requires an individual try/catch since each field is treated individually

**Comment by Namdrib** [comment]
Current behaviour is to just abort changing the dififculty

**Comment by Namdrib** [comment]
Fixed by 9db1c0f
